### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by adityamangal1**
.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ further defined and clarified by project maintainers.
 
 ### Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
+Instances of abusive, harassing, or otherwise, unacceptable behavior may be
```
Comment (M4cs) [inline CONTRIBUTING.md:71]: This is actually not a typo

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ Please note we have a code of conduct, please follow it in all your interactions
...
+3. Create a detailed pull request with information about what you've added.
 4. If a maintainer requests any changes, make them in the same branch and commit to your PR.
 
 ## Code of Conduct
 
 ### Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
+contributors and maintainers pledge to make participation in our project and
```
Comment (M4cs) [inline CONTRIBUTING.md:20]: This is not a typo

Comment (adityamangal1) [reply CONTRIBUTING.md:20]: See line 3

Comment (adityamangal1) [reply CONTRIBUTING.md:20]: Please see line 3

**Pull request closed without merging**